daguerre/models.py

```diff
@@ -99,17 +109,57 @@ def delete_adjusted_images(sender, **kwargs):
     qs.delete()
 
 
+def upload_to(instance, filename):
+    """
+    Construct the directory path where the adjusted images will be saved to
+    using the MD5 hash algorithm.
+
+    Can be customized using the DAGUERRE_PATH setting set in the project's
+    settings. If left unspecified, the default value will be used, i.e. 'dg'.
+    WARNING: The maximum length of the specified string is 13 characters.
+
+    Example:
+    * Default: dg/ce/2b/7014c0bdbedea0e4f4bf.jpeg
+    * DAGUERRE_PATH = 'img': img/ce/2b/7014c0bdbedea0e4f4bf.jpeg
+
+    Known issue:
+    * If the extracted hash string is 'ad', ad blockers will block the image.
+      All occurrences of 'ad' will be replaced with 'ag' since the MD5 hash
+      produces letters from 'a' to 'f'.
+    """
+
+    first_dir = None
+    if hasattr(settings, 'DAGUERRE_PATH'):
+        first_dir = settings.DAGUERRE_PATH
+
+    if not first_dir or len(first_dir) > 13:
+        msg = ('The DAGUERRE_PATH value is more than 13 characters long! '
+               'Falling back to the default value: "dg".')
+        warnings.warn(msg)
```

Comment:
This warning is should only be sent if first_dir is actually too long - currently, sent in that case _and_ if it's a falsey value.